Cython/Utility/Exceptions.c

```diff
@@ -1013,7 +1013,7 @@ static void __Pyx_AddTraceback(const char *funcname, int c_line,
 
 ///////////////////////////// FloatExceptionCheck.proto ///////////////////////////
 
-// Detect is error_value is NaN, and use a different check in that case
+// Detect if error_value is NaN, and use a different check in that case
 #define __PYX_CHECK_FLOAT_EXCEPTION(value, error_value) \
```

Comment:
There might be an issue with `error_value` being evaluated more than once though (given it's potentially an unknown macro)

Reply:
Using something non-trivial as (fast) error value should be rather unattractive for users, so I doubt that it'd be an issue in practice.